docs/developer_guide/playbooks.md

```diff
@@ -13,12 +13,58 @@ order they are supposed to be used.
 > `development` respectively. **In all cases, the `-e "customer=foo env_type=bar"` option is not required if you want to work with `patient0` in
 > `development`**.
 
+## `delete_project.yml`
+
+This playbook delete a project with all OpenShift objects for a customer
+(default: `patient0`) in a particular environment (default: `development`).
+
+### Usage
+
+```bash
+# development
+$ bin/ansible-playbook delete_project.yml -e "customer=patient0 env_type=staging"
+
+# native command for production
+$ docker run --rm -it \
+    --env-file env.d/production \
+    arnold \
+    ansible-playbook delete_project.yml -e "customer=patient0 env_type=staging"
+```
+
+## `bootstrap.yml`
+
+This playbook is a "meta" playbook that creates a new project with all required
+OpenShift objects for a customer (default: `patient0`) in a particular
+environment (default: `development`).
+It creates also the first deployement.
```

Comment:
I would remove this sentence to avoid repetition.

Reply:
done